packages/web-components/src/components/footer/footer-composite.ts

```diff
@@ -242,20 +257,81 @@ class DDSFooterComposite extends ModalRenderMixin(
     `;
   }
 
-  renderLightDOM() {
+  renderLanguageSelector(slot = 'language-selector') {
     const {
-      buttonLabel,
       clearSelectionLabel,
-      disableLocaleButton,
-      langDisplay,
+      _isMobile: isMobile,
       langList,
       languageSelectorLabel,
       selectedLanguage,
       size,
+    } = this;
+    return isMobile
+      ? html`
+          <dds-language-selector-mobile
+            size="${size || DROPDOWN_SIZE.EXTRA_LARGE}"
```

Comment:
Thanks @jkaeser! I think the size prop is actually unessary here, since it is already being set in the component: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/fcaa459c82b197b760e589c1378e4b3c1041058b/packages/web-components/src/components/footer/language-selector-mobile.ts#L34

```suggestion
```

Reply:
Ah, there's a discrepancy: the language selector components would make it seem like they should always render with the XL size, but that's not actually true. The micro version of the footer uses a sort of hacked "micro" size dropdown. I've pushed changes to hopefully make this a little more clear.